tests/phpunit/tests/comment.php

```diff
@@ -271,6 +271,91 @@ public function test_get_approved_comments_with_post_id_0_should_return_empty_ar
 		$this->assertSame( array(), $found );
 	}
 
+	/**
+	 * @ticket 53962
+	 */
+	public function test_can_reply_to_approved_comment() {
+		// Must be set for `get_comment_id_fields()`.
+		$_GET['replytocom'] = $this->create_comment_with_approval_status( true );
+
+		$expected  = "<input type='hidden' name='comment_post_ID' value='" . self::$post_id . "' id='comment_post_ID' />\n";
+		$expected .= "<input type='hidden' name='comment_parent' id='comment_parent' value='" . $_GET['replytocom'] . "' />\n";
+		$actual    = get_comment_id_fields( self::$post_id );
+
+		$this->assertSame( $expected, $actual );
+	}
+
+	/**
+	 * @ticket 53962
+	 */
+	public function test_cannot_reply_to_unapproved_comment() {
+		// Must be set for `get_comment_id_fields()`.
+		$_GET['replytocom'] = $this->create_comment_with_approval_status( false );
+
+		$expected = "<input type='hidden' name='comment_post_ID' value='" . self::$post_id . "' id='comment_post_ID' />\n";
```

Comment:
`$expected` will need to change for no-reply version to match my suggestion above.

:1234: Applies to multiple tests.

Reply:
I'm going to update the PR based on this, add some additional tests and tidy up the test names.